src/components/UNSTABLE__Shell/PopoverAccountItem/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @file Popover account item.
+ * @copyright IBM Security 2020
+ */
+
+import { func, node, string } from 'prop-types';
+import React from 'react';
+
+import { appendComponentNamespace } from '../../../globals/namespace';
+
+import { namespace as accountNamespace } from '../PopoverAccount';
+
+const namespace = appendComponentNamespace(accountNamespace, 'account');
+
+function PopoverAccountItem({ children, id, onClick, ...other }) {
```

Comment:
For `id` can we utilize Carbon's instance id generation tool? (used in `PanelV2` here: https://github.com/carbon-design-system/ibm-security/blob/dev/src/components/PanelV2/PanelV2.js#L28)

Reply:
This needs to be a consumer-supplied value as it signifies a user account ID, similarly to the previous implementation

Reply:
To your point, though, there are places like `HeaderActions` and `ShellToolbar` that could benefit from this!